### ISSUE TITLE
Texts: Remove "software" from "the Jamulus software"

### ISSUE THREAD

**Issue by hoffie** [jamulussoftware/jamulus#2350]
**Describe the task**

Lots of texts within Jamulus' UI (probably mainly What's This / tooltips) contain descriptions like "This means X in *the Jamulus software*". At least to me, this sounds unnatural, especially when translated.
Should we shorten this to "This means X in *Jamulus*"?

Initial discussion by @ann0see and me here: https://github.com/jamulussoftware/jamulus/pull/2349#discussion_r800097067

**Comment by ann0see** [comment]
Yes. I'd update it. Probably it's a good first issue (it can be done easily with grep)?

**Comment by gilgongo** [comment]
I'd go further and simply remove the mention of Jamulus, since it should be obvious what software you are looking at or using when you use What's This / tooltips. It's just possible that in some some contexts this would produce an ambiguity though (eg when talking about ASIO perhaps), in which case we'd need to keep "in Jamulus".

**Comment by gilgongo** [comment]
BTW I was thinking of a comprehensive review of all UI copy once the website had settled down (since the two are related), but there's no harm in making the odd small change meanwhile. Every time I think the website has finally settled though, there's yet another huge round of changes to be made and 3.8.9 will be no exception. Once that's out though, perhaps we can look at the app.

**Comment by jujudusud** [comment]
Is there another place where "software" has to be changed ?

**Comment by ann0see** [comment]
I didn't find any other places, but you can grep the files probably?

**Comment by gilgongo** [comment]
I didn't find any either. Closing in the interests of keeping the number of issues down.
